R/install.R

```diff
@@ -27,6 +28,17 @@ install_azureml <- function(version = "1.0.74",
     stop("Anaconda not installed or not in system path.")
   }
 
+  # set envname based on reticulate version
+  if (is.null(custom_envname)) {
+    if (utils::packageVersion("reticulate") >= "1.14") {
+      envname <- "r-reticulate"
+    } else {
+      envname <- "r-azureml"
+    }
+  } else {
+    envname <- custom_envname
+  }
+
```

Comment:
In the case that reticulate == 1.14, will we also have to specify the python dependencies (e.g. azureml-sdk) in the DESCRIPTION file so that the default created r-reticulate environment will install it? "Declaring a Python Dependency" section of https://blog.rstudio.com/2019/12/20/reticulate-1-14/